client/src/components/Card/NeedCard.tsx

```diff
@@ -7,47 +7,34 @@ import Typography from '@mui/material/Typography';
 import LocationOnIcon from '@mui/icons-material/LocationOn';
 import theme from '../../theme';
 import Example from '../../assets/need-example.png';
-import { Tooltip } from '@mui/material';
 
 type Props = {
   title: string;
   type: 'need' | 'donation';
   sx?: object;
   datePosted: Date;
   poster: any;
-  org?: string;
+  org: string;
   description: string;
   condition: string;
   location: string;
   imgUrls?: string[];
   children?: ReactNode | ReactNode[];
 };
 
-const ellipsesStyle = {
-  whiteSpace: 'nowrap',
-  textOverflow: 'ellipsis',
-  width: '100%',
-  boxSizing: 'border-box',
-  overflow: 'hidden',
-};
-
 export default function NeedCard(props: Props) {
   const image = props.imgUrls?.length ? props.imgUrls[0] : Example;
   return (
-    <Card
-      title={props.title}
-      type={props.type}
-      datePosted={props.datePosted}
-      sx={props.sx || {}}
-      org={props.poster.firstName}
-    >
-      <CardContent>
-        <Box sx={{ marginTop: theme.spacing(1), padding: '0.5em' }}>
-          <Tooltip title={props.title}>
-            <Typography variant="h1" sx={ellipsesStyle}>
-              {props.title}
-            </Typography>
-          </Tooltip>
```

Comment:
I made these changes earlier, they should not be reverted. The title should not be hardcoded as "Blazers", and the title should take up only 1 line, with a tooltip to indicate the full title.  The terms "apparel" and "long term"  shouldn't be hardcoded either.  We can wrap these in some conditional logic to show {asset.category} and {asset.urgency}  if present, but I felt having all this placeholder text makes it look like we support features we don't support, and makes it harder to determine what work still needs to be done. That's why I left those for a separate PR adding those features in.  If you want placeholders though, can we do something like "{category}" "{urgency}", so thats it clear we need to work on those?

Reply:
All of this make sense! Thanks for changing it and the explanation!

Reply:
@alexislcstone  are all the comments addressed? Is this ready to be closed and merge?

Reply:
@garimacha I've taken over this feature branch, addressed comments and finalized where Alexis has paused her work.

Reply:
Thank you @EtoKruto is there any other ticket you are also working on? I need details to update my planning document. Thanks.